ted_sws/notice_validator/services/__init__.py

```diff
@@ -1 +1,25 @@
+from typing import List
+
+from ted_sws.core.model.notice import Notice
+from ted_sws.core.model.validation_report import ReportNotice
+from ted_sws.core.model.validation_report_data import ReportNoticeData
+
 NOTICE_IDS_FIELD = "notice_ids"
+
+
+def transform_report_notice(report_notice: ReportNotice) -> ReportNoticeData:
```

Comment:
Why this functions is in ___init__.py ? I recommend to move in specific file for this scope.

Reply:
As good implementation will be a constructor method in ReportNoticeData class from ReportNotice object